docs/index.html

```diff
@@ -327,6 +327,15 @@ <h6 class="text-info">
               Only links clicked <strong>outside</strong> of browsers will open
               Browserosaurus. This is by design.
             </p>
+            <h6 class="text-info">
+              I would like to open a link in private/incognito mode / in the background.
+              Can I do this?
+            </h6>
+            <p>
+              Hold down the <kbd>SHIFT</kbd> key while choosing the browser to open it
+              in a private/incognito window. Hold the <kbd>ALT</kbd> key in order to
+              open the link in the background.
+            </p>
```

Comment:
Let's split this in two:

###### How do I open a link in the background?

Hold the <kbd>ALT</kbd> key when selecting a browser.

###### How do I open a link in private/incognito mode?

Hold the <kbd>SHIFT</kbd> key when selecting a browser.